site2/website/release-notes.md

```diff
@@ -1,6 +1,438 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.9.0 &mdash; 2021-11-25 <a id=“2.9.0”></a>
+
+### News and noteworthy
+- PIP-45 Pluggable metadata interface introduced many changes about ZooKeeper metadata management: consistency, resilience, stability, tech debt reduction (less code duplication)
+- Pulsar IO: Oracle Debezium connector, new Schema aware Elasticsearch sink connector
+- Many improvements to the Pulsar clients, with PIP-83, PIP-91, PIP-96 (see below)
+- Geo-replication improvements: PIP-88 replicate schemas across clusters
+- Apache Kafka connect sinks can be run as Pulsar sinks [#9927](https://github.com/apache/pulsar/pull/9927)
+
+### Breaking changes
```

Comment:
What's the difference between `News and noteworthy` and `Breaking changes`? Can we put them into one section?

Reply:
This is very common, I saw this in many other projects and products:

- `News and noteworth`: cool stuff, only good and nothing bad
- `Breaking changes`: this is usually painful for people who upgrade

So it is better to keep these sections separated